lasio/las.py

```diff
@@ -279,68 +300,31 @@ def read(
                             traceback.format_exc()[:-1]
                             + " in data section beginning line {}".format(i + 1)
                         )
-                    logger.debug(
-                        "Read ndarray {arrshape} from data section".format(
-                            arrshape=arr.shape
-                        )
-                    )
-
-                    # This is so we can check data size and use self.set_data(data, truncate=False)
-                    # in cases of data.size is zero.
-                    data = arr
-
-                    if data.size > 0:
-                        # TODO: check whether this treatment of NULLs is correct
-                        logger.debug("~A data {}".format(arr))
-                        if version_NULL:
-                            arr[arr == provisional_null] = np.nan
-                        logger.debug("~A after NULL replacement data {}".format(arr))
-
-                        # Provisionally, assume that the number of columns represented
-                        # by the data section's array is equal to the number of columns
-                        # defined in the Curves/Definition section.
 
-                        n_columns_in_arr = len(self.curves)
+                    # Assign data to curves.
+                    n = 0
```

Comment:
Could the name of the variable 'n' be changed to something more descriptive?  Something like curve_array_idx.  This helps others understand the code and helps with searching for the variable.

Reply:
Thanks, good point, I've renamed.